src/main/common/utils/numericUtils.ts

```diff
@@ -14,3 +14,37 @@ export function toNumberOrUndefined (value: any): number {
 
   return isNaN(numberVal) ? undefined : numberVal
 }
+
+export function containsAValidComma (input: string): boolean {
+  const output: string[] = input.split(',')
+
+  return output[output.length - 1].length >= 3
+}
+
+export function toAValidFloatOrOriginalValue (input: string): any {
+  const output: string = this.containsAValidComma(input) ? input.replace(new RegExp(/,/g), '') : input
+
+  const numberValue = parseFloat(output)
+
+  if (!output.includes(',') && !isNaN(numberValue) && !this.isExponential(numberValue)) {
+    return numberValue
+  } else {
+    return output
+  }
+}
+
+export function isExponential (numberValue: number) {
+  return numberValue.toString().match(new RegExp('\\d*[Ee][+-]?\\d*?'))
+}
+
+export function toAValidNumberOrOriginalValue (input: string): any {
+
+  const output: string = this.containsAValidComma(input) ? input.replace(new RegExp(/,/g), '') : input
```

Comment:
I think you can just use the regular expression literal `/,/` instead of `new RegExp(/,/g)`.

Reply:
I just check and the `g` switch is needed, so `/,/g` would suffice (no need to call `new RegExp`).